package.json

```diff
@@ -59,8 +59,8 @@
     "build:production": "tsup --env.NODE_ENV=production",
     "clean": "rm -rf dist",
     "clean:mars-api": "rm -rf .mars-api-temp && rm -rf packages/heyapi/src",
-    "example:iife": "npm run build && cd examples/iife && npm install && npm run dev",
-    "example:react": "npm run build && cd examples/react && npm install && npm run dev",
+    "example:iife": "npm run build:production && cd examples/iife && npm install && npm run dev",
+    "example:react": "npm run build:production && cd examples/react && npm install && npm run dev",
```

Comment:
Can we add `example:iife:staging` and `example:react:staging` scripts here to be able to also explicitly run the examples against staging (not as the default)